rotkehlchen/globaldb/handler.py

```diff
@@ -984,6 +986,86 @@ def add_single_historical_price(entry: HistoricalPrice) -> bool:
 
         return True
 
+    @staticmethod
+    def add_manual_current_price(
+            from_asset: Asset,
+            to_asset: Asset,
+            price: Price,
+    ) -> None:
+        """Adds manual current price and turns previously known manual current price into a
+        historical manual price.
+        May raise:
+        - InputError if some db constraint was hit. Probably means manual price duplication."""
+        with GlobalDBHandler().conn.write_ctx() as write_cursor:
+            try:
+                write_cursor.execute(
+                    'UPDATE price_history SET source_type=? WHERE source_type=? AND from_asset=?',
+                    (
+                        HistoricalPriceOracle.MANUAL.serialize_for_db(),
+                        HistoricalPriceOracle.MANUAL_CURRENT.serialize_for_db(),
+                        from_asset.identifier,
+                    ),
+                )
+            except sqlite3.IntegrityError as e:
+                # if we got an error, there has to be a corresponding entry in the db
+                with GlobalDBHandler().conn.read_ctx() as cursor:
+                    timestamp = cursor.execute(
+                        'SELECT timestamp FROM price_history WHERE source_type=? AND from_asset=?',
+                        (
+                            HistoricalPriceOracle.MANUAL_CURRENT.serialize_for_db(),
+                            from_asset.identifier,
```

Comment:
We do an extra query on case of error to show useful information to the user about how to fix the error. Maybe put that in the code